amd_openvx/openvx/ago/ago_interface.cpp

```diff
@@ -1762,11 +1807,47 @@ int agoInitializeGraph(AgoGraph * graph)
 	{
 		AgoKernel * kernel = node->akernel;
 		vx_status status = VX_SUCCESS;
+		// handle reverification path
+		vx_bool first_time_verify = ((graph->verified == vx_false_e) && (graph->reverify == vx_false_e)) ? vx_true_e : vx_false_e;
+		if (kernel->user_kernel) {
+			if (!first_time_verify) { //re-verify
+				if(kernel->deinitialize_f) {
+					if(node->local_data_set_by_implementation == vx_false_e) {
+						node->local_data_change_is_enabled = vx_true_e;
+					}
+					status = kernel->deinitialize_f(node, (vx_reference *)node->paramList, node->paramCount);
+					node->local_data_change_is_enabled = vx_false_e;
+					if (status != VX_SUCCESS) {
+						graph->reverify = vx_false_e;
+						graph->verified = vx_true_e;
+						graph->state = VX_GRAPH_STATE_VERIFIED;
+					}
+					else {
+						graph->reverify = vx_true_e;
+						graph->verified = vx_true_e;
+						graph->state = VX_GRAPH_STATE_VERIFIED;
+					}
+				}
+				if (node->localDataSize == 0) {
+					if(node->localDataPtr) {
+						if(!first_time_verify && node->localDataPtr)
```

Comment:
Codacy found an issue: [Condition '!first_time_verify' is always true](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6287409)